sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/models/ContainerRegistriesCreateManifestHeaders.java

```diff
@@ -30,8 +29,6 @@ public final class ContainerRegistriesCreateManifestHeaders {
     @JsonProperty(value = "Location")
     private String location;
 
-    private static final HttpHeaderName DOCKER_CONTENT_DIGEST = HttpHeaderName.fromString("Docker-Content-Digest");
```

Comment:
Something doesn't seem right with code generation here, it should be creating `HttpHeaderName`s and using those below to access the header value.

Reply:
yep, thanks, I updated swagger and it looks good now